Assets/Player.cs

```diff
@@ -11,19 +11,26 @@ public class Player : MonoBehaviour
     void Awake()
     {
         controls = new InputMaster();
-        controls.GamePlay.Movement.performed += ctx => move = ctx.ReadValue<Vector2>();
+        controls.GamePlay.Movement.performed += HandleMovement;
         controls.GamePlay.Movement.canceled += ctx => move = Vector2.zero;
     }
 
+    void HandleMovement(InputAction.CallbackContext context)
+    {
+        move = context.ReadValue<Vector2>();
+        // Debug.Log($"Moving {move}");
+    }
+
     void Update()
     {
-        Vector2 m = new Vector2(move.x, move.y) * Time.deltaTime;
-        transform.Translate(m, Space.World);
+        Vector3 m = new Vector3(move.x, 0, move.y) * Time.deltaTime;
+        transform.Translate(m);
     }
 
     void OnEnable()
     {
         controls.GamePlay.Enable();
+        controls.GamePlay.Movement.performed += HandleMovement;
```

Comment:
I'm not sure if Awake or OnEnable are the 'proper' ways to do this, I just saw all the tutorials using OnEnable so I went for redundant.

We should condense to one or the other, depending on what is the healthier choice